### ISSUE TITLE
Dental panoramic detection: Overfitting problem

### ISSUE THREAD

**Issue by gksruf** [WongKinYiu/ScaledYOLOv4#281]
Hello. I am not an expert in deep learning and seeking for an help. I am trying to customize YOLOv4 with two pathologic lesions on dental panoramic radiographs. Class 1 has total of 455 (384 training set & 71 test set) and Class 2 has total of 1441 images (1223 training set & 218 test set). All dental panoramic images seems similar in general because people have similar bone structure. However, two different lesions are usually located in the apex of the tooth and they are both pretty small in the large panoramic radiograph. Since labeled lesions are small, we cropped the image when training and tested them. However, the resulting graph looks like below:

![image](https://user-images.githubusercontent.com/12556799/123534077-5567f380-d755-11eb-9863-342174c0cf2b.png)

It overfits very soon. What should I do to fix this problem? Should I try to change masks, anchors, classes, or num in "yolov4.cfg"? Would it help improve the problem? Especially, I am thinking about changing anchors and I wonder how.
Or should I use YOLOv3 instead of v4? or could I change v4 more shallower using darknet? Please help me solve the problem. Thank you and have a great day!

**Comment by huyhoangle86** [comment]
Do you have dev set or validation set ?

**Comment by xcc-1921** [comment]
I am poor at English.
You may change the 'batch_size' in 'utils/dataset' or training cmd.(batch_size>1)(16 or 64 maybe)
The lower it is,the result may have problem.

**Comment by xcc-1921** [comment]
I am poor at English.
You may change the 'batch_size' in 'utils/dataset' or in the training cmd.(batch_size>1 ,16 or 64 maybe)
The lower it is,the result may have problem.

**Comment by xcc-1921** [comment]
Some training model of yolov4 design for little object.
You can use other ‘.cfg’ or ‘weights’.（I can't remember which cfg or weights）
